Library/Homebrew/rubocops/text.rb

```diff
@@ -62,6 +62,10 @@ def audit_formula(formula_nodes)
             problem %q(use "xcodebuild *args" instead of "system 'xcodebuild', *args")
           end
 
+          if method_called_ever?(body_node, :xcodebuild) && !depends_on?(:xcode)
```

Comment:
Need to reverse condition to get correct offensive node
```suggestion
          if !depends_on?(:xcode) && method_called_ever?(body_node, :xcodebuild)
```